### DISCUSSION TITLE
Request output at very increments?

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#2326] [Q&A]
Hello, all

I have a basic question for Radioss? How can I request output at each increments? not the time step. I need the outputs for each incements.

Thanks for helping

Bests
Lee

**Comment by MarianBulla** [comment]
Hello li2qn,
if you are in the explicite domain, your "increments" are the cycles (1 cycle is 1 time-step).

So, fix the time-step to a lower value as the critical timestep, using "/DTIX" keyword in the engine file, 
and use the same frequency for your output.

This will give you the output at each cycle (= time-step = "increment").

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Hello Marian,

Thanks for your help. Please if the engine file will give me output for each cycles? and force the output for each cycle at the smallest time increment 0.9731E-04?
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/efef0503-d334-41a6-834d-7f5ae35cc914)

Bests
lee

**Reply by MarianBulla** [reply]
Hello Lee,

yes, then you just need:

.
.
/DTIX
0.9e-4 0.9e-4
/ANIM/DT
0.0 0.9e-4
.
.
(please, comment-out the /DT/NODA/CST. You don't need it):
#/DT/NODA/CST
#0.9 1e-7

Hope it helps you ?

Thanks and best 
Marian

**Reply by li2qn** [reply]
Thanks so much. That helps for sure.

**Reply by MarianBulla** [reply]
You're always welcome !  👍
